src/snmalloc/mem/localalloc.h

```diff
@@ -184,6 +184,13 @@ namespace snmalloc
       }
 
       return check_init([&](CoreAlloc* core_alloc) {
+        if (size > bits::one_at_bit(bits::BITS - 1))
```

Comment:
I guess I understand not initializing the frontend in response to an absurd request, but this seems like the kind of thing that will be pretty rare?

Reply:
So `check_init` does always calls the supplied lambda, not just in the init case.

https://github.com/microsoft/snmalloc/blob/9d4466093a7c42e4fe43e032aeca356674d6e55c/src/snmalloc/mem/localalloc.h#L103-L111

This is dealing with this case off the fast path, and then setting the error.